src/seedu/addressbook/data/AddressBook.java

```diff
@@ -68,6 +71,13 @@ public UniquePersonList getAllPersons() {
         return new UniquePersonList(allPersons);
     }
 
+    /**
+     * Sorts all persons in the address book and returns a new UniquePersonList of all persons
```

Comment:
Does it return any value? You set the return type as void, so maybe the description of this method is off?